package.json

```diff
@@ -5,9 +5,10 @@
   "main": "index.js",
   "scripts": {
     "snapshot": "TAP_SNAPSHOT=1 tap test/*.js",
-    "test": "tap test/*.js --cov",
+    "test": "c8 --reporter=html --reporter=text tap test/*.js",
     "posttest": "standard",
-    "release": "standard-version"
+    "release": "standard-version",
+    "coverage": "c8 --reporter=text-lcov tap test/*.js | coveralls"
```

Comment:
Sorry if I'm missing context here, but is there a `c8 report` instead of having to run all the tests again?

Reply:
I haven't hammered out c8 report yet, but I'd be happy to do so before landing this.